internal/executor/instance/persistentworker/isolation/parallels/clone.go

```diff
@@ -56,14 +44,5 @@ func cloneFromDefault(ctx context.Context, vmNameFrom string) (*VM, error) {
 		return nil, fmt.Errorf("%w: failed to clone VM %q: %q", ErrVMFailed, vm.Ident(), firstNonEmptyLine(stderr))
 	}
 
-	if err := vm.isolate(ctx); err != nil {
```

Comment:
Should it set `delayedIsolation`?

Reply:
It's already set here: https://github.com/cirruslabs/cirrus-cli/pull/278/files#diff-ad16c76192d04ea42699ba6c12657f05dff37e2e2df1972197ab29b93b04305aR16.

Reply:
It's `cloneFromDefault` 🤔

Reply:
Ahh, you don't need to set `delayedIsolation` for `cloneFromDefault` because it's not delayed there (VM is not suspended and the isolation can be set before the VM is started, that's exactly how it's worker before).